governance/votes/Executive vote - March 25, 2022.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Template - [Executive Vote] Authorizing the ESM on DssFlash and Support for TUSD Implementation - March 25, 2022
+summary: Authorizing the Emergency Security Module on DssFlash to revoke governance control after Emergency shutdown and Adding Support for the New TUSD Implementation.
+date: 2022-03-25T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Support for TUSD Implementation and Authorizing the ESM on DssFlash - March 25, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The [Emergency Shutdown Module](https://makerdao.world/en/learn/governance/emergency-shutdown) will be authorized to revoke Governance control over the [Flash Mint Module](https://manual.makerdao.com/module-index/module-flash-mint-module) after an emergency shutdown.
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- Support for the new TUSD implementation contract ([0xd8D59c59Ab40B880b54C969920E8d9172182Ad7b](https://etherscan.io/address/0xd8d59c59ab40b880b54c969920e8d9172182ad7b)), which will allow collateral operation of TUSD-A to resume.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Emergency Shutdown Authorization
```

Comment:
"Authorizing ESM on DssFlash to remove Governance Permissions"?

Reply:
Good call